columnq/Cargo.toml

```diff
@@ -41,11 +41,6 @@ calamine = "0.19.1"
 
 tokio = { version = "1", features = ["rt-multi-thread"] }
 futures = "0.3"
-# S3
```

Comment:
now s3 is refactor, let me create a release PR for `0.8.2` after this PR is approved and merged.

after we adds GCS and Azure support, I think we can release `0.9.0`?

Reply:
that sounds good to me, let's do it!